public/index.html

```diff
@@ -24,6 +24,9 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Pangolin&family=Roboto+Slab:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
```

Comment:
- [ ] Kindly adhere to the requirement carefully, as per the requirement, you should follow the guidelines of the project which says, should follow the `Style` guidelines. But it seems like you are not following the correct `font-family`, `font-weight` and `font-size`. Following the guidelines will help you to be a detail-oriented developer.

![Screenshot from 2023-09-23 18-55-58](https://github.com/BiancaMe/johto-league/assets/66476329/24ebf090-f959-4f9e-a7c0-a6711c4a44c0)